pokemon_service/models/pokemon.go

```diff
@@ -0,0 +1,9 @@
+package model
+
+// Pokemon is a struct that defines the properties of a Pokemon
+type Pokemon struct {
```

Comment:
> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".
> 
> This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId".

https://github.com/golang/go/wiki/CodeReviewComments#initialisms